server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SignOffAuditRound.java

```diff
@@ -208,35 +209,30 @@ public String endpointBody(final Request request,
               cdb.ballotsAudited()));
 
           if (cdb.allAuditsComplete()) {
-            LOGGER.debug
-              (String.format
-               ("[signoff: RISK_LIMIT_ACHIEVED for %s County: "
-                + " cdb.estimatedSamplesToAudit()=%d,"
-                + " cdb.auditedSampleCount()=%d,"
-                + " cdb.ballotsAudited()=%d]",
-                cdb.county().name(),
-                cdb.estimatedSamplesToAudit(),
-                cdb.auditedSampleCount(),
-                cdb.ballotsAudited()));
             my_event.set(RISK_LIMIT_ACHIEVED_EVENT);
-            cdb.endAudits();
-            audit_complete = true;
+            final List<ComparisonAudit> terminated = cdb.endSingleCountyAudits();
+            LOGGER.debug(String.format("[signoff: all targeted audits finished in %s County."
+                                       + " Terminated these audits: %s]",
+                                       cdb.county().name(), terminated));
+            auditComplete = true;
           } else if (cdb.cvrsImported() <= cdb.ballotsAudited()) {
-            LOGGER.debug("[signoff: there are no more ballots in the county]");
-            my_event.set(BALLOTS_EXHAUSTED_EVENT);
-            cdb.endAudits();
-            audit_complete = true;
+            final List<ComparisonAudit> terminated = cdb.endSingleCountyAudits();
```

Comment:
In this case, we'd be terminating targeted and opportunistic single county audits.

Reply:
this and your other note might make good inline comments